exe/IHP/CLI/run-script

```diff
@@ -28,10 +28,11 @@ fi
 
 echo "import IHP.ScriptSupport" >> "$GHCI_SCRIPT"
 echo ":set -i." >> "$GHCI_SCRIPT"
+echo ":l Config/Config.hs" >> "$GHCI_SCRIPT"
 echo ":l Config/Config.hs Application/Script/$TASK_MODULE.hs" >> "$GHCI_SCRIPT"
 echo "import qualified Application.Script.$TASK_MODULE as Script" >> "$GHCI_SCRIPT"
-echo "IHP.ScriptSupport.runScript Script.run" >> "$GHCI_SCRIPT"
-echo ":quit" >> "$GHCI_SCRIPT"
+echo "let ?frameworkConfigIO = config in IHP.ScriptSupport.runScript Script.run" >> "$GHCI_SCRIPT"
```

Comment:
We also need to update this make task here:

https://github.com/digitallyinduced/ihp/blob/master/lib/IHP/Makefile.dist#L179

It builds a script to a binary. You can run it via `make build/bin/Script/SomeScript` :)

Reply:
It might also be a little bit deceiving that the configuration does not change for scripts that are already built. We should either add a good warning about this in the guide or think about a way to dynamically load the config.

Reply:
https://purelyfunctional.org/posts/2016-05-20-dynamic-loading-haskell-module.html It is possible, but it's not pretty :/

Reply:
> It might also be a little bit deceiving that the configuration does not change for scripts that are already built

I would be ok with that. Scripts are usually only build to binary when running in productions (in dev you just call them using `./Application/Script/SomeScript.hs`).